libkbfs/prefetcher.go

```diff
@@ -59,21 +88,21 @@ func (p *blockPrefetcher) run() {
 	for {
 		select {
 		case req := <-p.progressCh:
-			if p.retriever == nil {
-				continue
-			}
 			ctx, cancel := context.WithCancel(context.Background())
-			ch := p.retriever.Request(ctx, req.priority, req.kmd, req.ptr, req.block, TransientEntry)
+			errCh := p.retriever.Request(ctx, req.priority, req.kmd, req.ptr, req.block, TransientEntry)
 			wg.Add(1)
 			go func() {
 				defer wg.Done()
+				defer cancel()
+				p.log.CDebugf(ctx, "Begin prefetch for block %s.", req.ptr.ID)
 				select {
-				case _ = <-ch:
+				case err := <-errCh:
+					p.log.CDebugf(ctx, "Done prefetch for block %s. Error: %v", req.ptr.ID, err)
```

Comment:
Probably want `%+v` to get error stack traces (here and anywhere else we print errors)